integrations-and-supported-tools/fastai/scripts/Neptune_fastai.py

```diff
@@ -0,0 +1,17 @@
+import fastai
+from neptune_fastai.impl import NeptuneCallback
```

Comment:
it should be added (and imported) from neptune-client

`from neptune.new.integrations.fastai import NeptuneCallback`